lib/DataFetcher.dart

```diff
@@ -219,4 +260,18 @@ class DataFetcher {
       ),
     ];
   }
-}
+
```

Comment:
These can be deleted since they were removed in 0bbf8cfad92f92243a2cb601ef176755a029e225.